src/components/about/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
```

Comment:
Is this part of the `pr` ?

Reply:
Woops, apologies they were stash changed for a potential resuable modal that could be used for an about/options modal for a one of the issues - have removed.